### ISSUE TITLE
Hide "Forgot password?" option until it is supported by API

### ISSUE THREAD

**Issue by bglamadrid** [trebol-ecommerce/ngx-trebol-frontend#79]
Discussed in [this comment](https://github.com/trebol-ecommerce/trebol-api/issues/16#issuecomment-922115379) is the fact that the frontend features a link to reset the password but basically does nothing yet.
Due to a fuzzy development from my part, that should stay hidden for now until it can serve a purpose through the API.

**Comment by bglamadrid** [comment]
This is actually not yet implemented; a new issue for it will be created after #78 is merged
